source/administration/command-line-tools.rst

```diff
@@ -435,14 +435,33 @@ mattermost import slack
       sudo ./mattermost import slack myteam slack_export.zip
 
 mattermost ldap
--------------
+----------------
 
   Description
     Commands to configure and synchronize AD/LDAP.
 
   Child Command
+    -  `mattermost ldap idmigrate`_ - Migrate the LDAP Id Attribute to a new value
     -  `mattermost ldap sync`_ - Synchronize now
 
+mattermost ldap sync
```

Comment:
@crspeller Is there any more concrete steps or information we can give to admins when doing the migration?

Reply:
I would be less confusing. We know what AuthData is, but no user is ever supposed to see that. 

I would write: "Run this utility to change the value of your ID Attribute without your users losing their accounts. After running the command you can change the ID Attribute to the new value in your config.json. For example if your current ID Attribute was sAMAccountName and you wanted to change it to objectGUID you would:
1. Wait for an off time when your users won't mind a server restart.
2. Run the command `mattermost ldap idmigrate objectGUID`
3. Edit your config.json and change your IdAttribute field to your new IdAttribute `objectGUID`
4. Restart the Mattermost server"

Reply:
Thanks! @crspeller Updated